app/src/main/res/layout/activity_main2.xml

```diff
@@ -29,6 +29,7 @@
         android:layout_gravity="bottom|end"
         android:layout_marginEnd="@dimen/fab_margin"
         android:layout_marginBottom="16dp"
-        app:srcCompat="@android:drawable/ic_dialog_email" />
+        app:srcCompat="@android:drawable/ic_dialog_email"
+        android:contentDescription="@string/todo" />
```

Comment:
Avoid TODO here.  Complete the implementation when you introduce the attribute.  TODO is essentially technical debt.